testsuite/features/step_definitions/command_steps.rb

```diff
@@ -703,7 +703,11 @@
 
 When(/^I wait until the package "(.*?)" has been cached on this "(.*?)"$/) do |pkg_name, host|
   node = get_target(host)
-  cmd = "ls /var/cache/zypp/packages/susemanager:test-channel-x86_64/getPackage/*/*/#{pkg_name}*.rpm"
+  if host == 'sle_minion'
```

Comment:
In order to cover other clients could you please do that?

```suggestion
  if host.include? 'ubuntu'
      cmd = "ls /var/cache/apt/archives/#{pkg_name}*.deb"
  else
      cmd = "ls /var/cache/zypp/packages/susemanager:test-channel-x86_64/getPackage/*/*/#{pkg_name}*.rpm"
  end
```

Reply:
I already quickly raise the issue with Marcelo. This is part of a port, so that change would need to happen in the other branches as well.

Marcelo?